shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/controllers/ManagedBridgeController.java

```diff
@@ -219,4 +168,12 @@ private String extractBrokerPath(KnativeBroker broker) {
             return null;
         }
     }
+
+    private void notifyManager(ManagedBridge managedBridge) {
```

Comment:
@vaibhavjainwiz See discussion about removing callback to Control Plane from Controllers and simply doing it on a timed loop in the `ManagerBridgeSyncService`

Reply:
done